pkg/i18n/english.go

```diff
@@ -706,9 +715,18 @@ func EnglishTranslationSet() TranslationSet {
 		UpdatesRejectedAndForcePushDisabled: "Updates were rejected and you have disabled force pushing",
 		LcCheckForUpdate:                    "check for update",
 		CheckingForUpdates:                  "Checking for updates...",
+		UpdateAvailableTitle:                "Update available!",
+		UpdateAvailable:                     "Download and install version {{.newVersion}}?",
+		UpdateInProgressWaitingStatus:       "updating",
+		UpdateCompletedTitle:                "Update completed!",
+		UpdateCompleted:                     "Update has been installed successfully. Restart lazygit for it to take effect.",
+		FailedToRetrieveLatestVersionErr:    "Failed to retrieve version information, please check the logs for details.",
```

Comment:
Not a 100% sure regarding wording / content of this one.

Reply:
Kept this as-is for now or did you mean for me to remove the "please check the logs for details" suffix?

Reply:
yeah I'd remove the suffix just because I don't think the logs necessarily will have the info for fixing the problem

Reply:
Ok, done :+1: